### PR TITLE
refactor(npm): expand on install, build and server tasks

### PR DESCRIPTION
**Pull request by bighappyface**
This PR adds a few convenience `npm` commands and leverages local installs for `grunt` and `bower`.

### CODE REVIEW

**Comment by bighappyface** [comment]
@brady-vitrano @SyntaxStacks thoughts?

File: package.json [hunk 1]
```diff
@@ -6,12 +6,17 @@
     "Brady Vitrano <brady.vitrano@rackspace.com>"
   ],
   "scripts": {
-    "test": "grunt test",
-    "build": "bundle install && npm install && bower install"
+    "test": "npm run grunt test",
+    "postinstall": "bundle install && npm run bower install",
+    "build": "npm run grunt build",
+    "server": "cd docs && jekyll server",
```
Comment (SyntaxStacks) [inline package.json]: one step closer.

**Comment by brady-vitrano** [comment]
Looks good to me

**Comment by bighappyface** [comment]
@brady-vitrano throw me collab status and I'll merge

**Comment by brady-vitrano** [comment]
@bighappyface Done!

**Pull request merged**